cli/npm/resolution/graph.rs

```diff
@@ -1329,40 +1465,30 @@ mod test {
     assert_eq!(
       packages,
       vec![
-        NpmResolutionPackage {
-          pkg_id: NpmPackageId::from_serialized("package-a@1.0.0").unwrap(),
+        TestNpmResolutionPackage {
```

Comment:
Below this point is mostly updates to use a new less verbose struct for testing. At the bottom of this file and line 3124 there are some new tests. I have commented where those are.